sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -210,6 +210,14 @@ private void startTracing(final @NotNull Activity activity) {
                 getAppStartDesc(coldStart),
                 appStartTime,
                 Instrumenter.SENTRY);
+
+        // in case we already have an end time, e.g. due to deferred SDK init we can already finish
+        // the span
+        final SentryDate appStartEndTime = AppStartState.getInstance().getAppStartEndTime();
+        if (appStartEndTime != null) {
+          appStartSpan.finish(SpanStatus.OK, appStartEndTime);
```

Comment:
M: `status` should only be set in case there's none otherwise you overwrite values set by the user, similar case on L345